server/src/handler/board.rs

```diff
@@ -73,12 +69,13 @@ fn get_space_id(req: &HttpRequest) -> Result<space::ID, Error> {
 pub async fn connect(
     req: HttpRequest,
     stream: web::Payload,
+    token: web::Query<Token>,
     spaces: web::Data<board::Registry>,
-    users: web::Data<user::Registry>,
+    user_client: web::Data<user::Client>,
 ) -> Result<HttpResponse, Error> {
-    let user = get_user(&req, users)?;
+    let user = get_user(token.tk, user_client).await?;
```

Comment:
QQ: We still expect that the token encodes the user ID in plain right? (well, at least for now). Do you still imgaine it possible the token could technically come from a cookie? Please correct me if I'm wrong :) Just trying to reason about `token.tk` here but the `user_id` signature ion line 45

Reply:
i'd like to move away form this is it really is bad practice and pretty much just some quick duct tape for the demo. we should have a proper signed token that can at a minimum map to a user (ideally a session). that said, we cannot reliably trust cookies in websocket upgrade requests. ideally we should move to this to some kind of first ping is an auth challenge, but in this case this has been moved to including it in the query string param.